salesapp.js

```diff
@@ -117,14 +148,27 @@ var runTableTimes = function(){
     }
   };
 };
+var runTimeTotals = function() {
+  var timeTotals = document.createElement('tfoot');
+  tableEl.appendChild(timeTotals);
+  var footTitle = document.createElement('th');
+  footTitle.textContent = 'Totals';
+  timeTotals.appendChild(footTitle);
+  for (var i = 0; i < hourlyTotalsStorage.length; i++) {
+    var fieldEl = document.createElement('td');
+    fieldEl.textContent = hourlyTotalsStorage[i];
+    timeTotals.appendChild(fieldEl);
+  }
+  // fieldEl.textContent = allTotal;
+  // timeTotals.appendChild(fieldEl);
+};
 
+//-----------------------------CALL FUNCTIONS--------------------------------------
 runTableTimes();
 runAllStoreData();
-
-
-var footerEl = document.createElement('tr');
-tableEl.appendChild(footerEl);
-
+runStoreHourTotals();
+runTimeTotals();
+sumEverything();
 for(var i = 0; i < storeArray.length; i++){
```

Comment:
You might want to break this out into it's own function too, like `drawAllStoreRows` or something.